pkg/sdkserver/sdkserver.go

```diff
@@ -547,43 +555,117 @@ func (s *SDKServer) stopReserveTimer() {
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTracking]
 func (s *SDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerID) (*alpha.Bool, error) {
-	panic("implement me")
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.Errorf("%s not enabled", runtime.FeaturePlayerTracking)
+	}
+	s.logger.WithField("PlayerId", id).Debug("Player Connected")
```

Comment:
Sorry Mark, one more
```suggestion
	s.logger.WithField("PlayerId", id.PlayerID).Debug("Player Connected")
```

Reply:
Good catch. Applying now!